app/src/main/java/com/onboarding/parkingsystemjava/mvp/view/ParkingView.java

```diff
@@ -0,0 +1,26 @@
+package com.onboarding.parkingsystemjava.mvp.view;
+
+import android.app.Activity;
+import android.widget.Toast;
+
+import com.onboarding.parkingsystemjava.R;
+import com.onboarding.parkingsystemjava.databinding.ActivityMainBinding;
+import com.onboarding.parkingsystemjava.mvp.contract.MainActivityContract;
+import com.onboarding.parkingsystemjava.mvp.view.base.ActivityView;
+
+public class ParkingView extends ActivityView implements MainActivityContract.MainActivityView {
+
+    private final ActivityMainBinding binding;
+
+    public ParkingView(Activity activity, ActivityMainBinding binding) {
+        super(activity);
+        this.binding = binding;
+    }
+
+    @Override
+    public void showParkingLotsAvailable(int parkingLots) {
+        if (getActivity() != null) {
+            Toast.makeText(getActivity(), getActivity().getString(R.string.main_activity_set_parking_toast_1) + " " + parkingLots + " " + getActivity().getString(R.string.main_activity_set_parking_toast_2), Toast.LENGTH_LONG).show();
```

Comment:
you could use a single String Resource , that acepts a int as a parameter.
Check this  out https://developer.android.com/guide/topics/resources/string-resource  (Formmatting strings)

Reply:
I think you need to use getContext(), you have it in the ActivityView

Reply:
you could use a single String Resource , that acepts a int as a parameter.
Check this out https://developer.android.com/guide/topics/resources/string-resource (Formmatting strings)